openapi/generator.go

```diff
@@ -758,6 +758,14 @@ func (g *Generator) newSchemaFromStructField(sf reflect.StructField, required bo
 	if t, ok := sf.Tag.Lookup(formatTag); ok {
 		schema.Format = t
 	}
+
+	// Set example value from tag to schema
+	if e := sf.Tag.Get("example"); e != "" {
```

Comment:
`sf.Tag.Get` is good enough in place of `Lookup` since we can juste skip if the value is empty, but we should use `strings.TrimSpace` before checking that `e != ""`.